mikupad.html

```diff
@@ -150,6 +157,30 @@
 #an-area {
 	min-height:3.5em;
 }
+.wi-textarea {
+	min-height: 4em;
+}
+
+#advancedContextPlaceholders {
+	background: var(--color-base-30);
+	margin: 1em auto;
+	padding: .25em;
+	border-radius: 2px;
+	width: fit-content;
+}
+#advancedContextPlaceholders th,
+#advancedContextPlaceholders td {
+	text-align: center;
+	padding: 0.15em;
+}
+#advancedContextPlaceholders td {
+	user-select:all;
+}
+
+#advanced-context-order-settings {
+	height: 3.5em;

```

Comment:
The issue seems to be solved by changing `height` to `min-height`.
![image](https://github.com/lmg-anon/mikupad/assets/139719567/ff8a14cd-1014-4187-9b2b-2a7d308bcd0a)

```suggestion
	min-height: 3.5em;
```

Reply:
> The context order textarea still looks broken to me, looks like a problem related to the CollapsibleGroup. Weird that this doesn't happen to you.

The wonders of CSS.
Anyways, thanks for the review. Suggestions committed.